src/routes/baseRoute.js

```diff
@@ -40,7 +41,10 @@ class Route {
         options: {
           allowUnknown: true
         },
-        payload: validator.formValidators,
+        // As of v19 Hapi no longer provides a default validation compiler. To fix this we use Joi.compile() which will
+        // either compile an uncompiled schema or return an already-compiled schema (which is the case with validators
+        // which return a Joi object, eg. NeedToConsultValidator)
+        payload: Joi.compile(validator.formValidators),
```

Comment:
Whilst the code is full of non-standard ways of implementing things, the drop-in way of re-applying Joi is to set it as the validator on the server, e.g. `server.validator(require('@hapi/joi'))`.

I think it would be better to go with that unless there's a particular reason not to.

Reply:
Good catch, I've updated `server.js` to do this.